### PR TITLE
SRE-139 Fix ansible-lint issues

### PR DESCRIPTION
**Pull request by hollow**

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -17,24 +17,12 @@ galaxy_info:
...
-    - elasticsearch_exporter
-    - kafka_exporter
-    - memcached_exporter
-    - mysqld_exporter
-    - node_exporter
-    - postgres_exporter
-    - process_exporter
-    - redis_exporter
-    - statsd_exporter
-    - windows_exporter
```
Comment (langerma) [inline meta/main.yml:38]: why did you delete these references? all this exporters are included.

**Review by langerma — CHANGES_REQUESTED** [review]

File: meta/main.yml (hunk 1 above)
Comment (hollow) [reply meta/main.yml:38]: these are invalid tags according to ansible-lint (no underscore allowed)

File: meta/main.yml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ---
 galaxy_info:
-  author: Markus Langer
```
Comment (hollow) [inline meta/main.yml:3]: btw, this was needed because otherwise ansible cannot compute the FQCN. we can also leave the author as-is and add a `namespace` instead

File: meta/main.yml (hunk 1 above)
Comment (langerma) [reply meta/main.yml:38]: okay 👍

File: meta/main.yml (hunk 2 above)
Comment (langerma) [reply meta/main.yml:3]: okay 👍

**Review by langerma — APPROVED** [review]
